libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -957,15 +965,15 @@ void NavEKF3::UpdateFilter(void)
     }
 
     if (primary != 0 && core[0].healthy() && !armed) {
-        // when on the ground and disarmed force the first lane. This
-        // avoids us ending with with a lottery for which IMU is used
-        // in each flight. Otherwise the alignment of the timing of
-        // the lane updates with the timing of GPS updates can lead to
-        // a lane other than the first one being used as primary for
-        // some flights. As different IMUs may have quite different
-        // noise characteristics this leads to inconsistent
+        // when on the ground and disarmed force the selected primary
+        // lane. This avoids us ending with with a lottery for which
```

Comment:
It's an existing issue and I'm being nitpicky but I think in general we should consolidate on either the word "core" or "lane" and try not to use them interchangeably.  They mean the same thing but it is confusing for non-english speakers to realise the two words mean the same thing.

Reply:
yep, fixed to use 'core' in this comment at least